src/ducks/components/App.jsx

```diff
@@ -43,12 +43,12 @@ export class App extends Component {
   }
 }
 
-const mapStateToProps = state => ({
+export const mapStateToProps = state => ({
```

Comment:
Why not isolate them into a module ? It feels more natural to me to override a module but I could be missing a point.